Assets/2_MainScene/S2_Scripts/S2_AsteroidScripts/S2_PoolController.cs

```diff
@@ -24,7 +24,10 @@ public class S2_PoolController : MonoBehaviour {
 
     protected int RNG, arrayIndex;
     protected float speed = 75.0f, waitTime = 1.8f, benchedTime = 5.4f;
-
+    public float GetWaitTime()
```

Comment:
Returnable should be placed with other returnable methods in script

Reply:
organization of my scripts is according to functionality, the method resting beneath the variable is best practice, can change, but makes navigating a script for someone more difficult if they are unaware of your methodology.  Perhaps we can put together a tech doc that covers all of the minor notes placed on merge?

Reply:
For this project specifically there is a script organization method already in use with all the other scripts, new scripts should follow the same methodology